images/router/haproxy/conf/haproxy-config.template

```diff
@@ -231,6 +231,29 @@ backend be_edge_http_{{$cfgIdx}}
   timeout server  {{$value}}
       {{ end }}
     {{ end }}
+
+{{ if index $cfg.Annotations "router.openshift.io/haproxy.rate-limit-connections" }}
```

Comment:
I followed the same format for annotations as Rajat ([line 222](https://github.com/JacobTanenbaum/origin/blob/c8747d9aa36a3c492dc9fbd724dcf285cfdf112f/images/router/haproxy/conf/haproxy-config.template#L222)). If I change mine the naming will be inconsistent, should I change Rajat's as well? @rajatchopra